### PR TITLE
New package: rpi-usbboot-20221104.153421

### PR DESCRIPTION
**Pull request by jcgruenhage**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

### CODE REVIEW

**Comment by jcgruenhage** [comment]
Huh, yeah, so this is going to need more Makefile patching for crosscompilation to work.

File: srcpkgs/rpi-usbboot/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'rpi-usbboot'
+_commit=1e651a88f0313a5ca50b12a9992dbce8930fe200
+pkgname=rpi-usbboot
+version=20220815.145439
+revision=1
+wrksrc="${pkgname/rpi-/}-${_commit}"
+build_style=gnu-makefile
+#make_dirs="/var/log/dir 0755 root root"
```
Comment (the-maldridge) [inline srcpkgs/rpi-usbboot/template:8]: Please remove commented lines.

**Review by the-maldridge — CHANGES_REQUESTED** [review]

File: srcpkgs/rpi-usbboot/template (hunk 1 above)
Comment (jcgruenhage) [reply srcpkgs/rpi-usbboot/template:8]: usually I remember to remove those, not sure why I forgot this time :D

**Comment by jcgruenhage** [comment]
Updated. Not stale.

**Pull request merged**